warrior/Actions/SeleniumActions/browser_actions.py

```diff
@@ -182,10 +182,8 @@ def browser_launch(self, system_name, browser_name="all", type="firefox",
                     output_dict[browser_fullname] = browser_inst
                     if "url" in browser_details and browser_details["url"]\
                             is not None:
-                        result, url = self.browser_object.check_url(browser_details["url"])
-                        if result == True:
-                            result = self.browser_object.go_to(url,
-                                                               browser_inst)
+                        self.browser_object.check_url(browser_details["url"])
```

Comment:
There is not point calling the function check_url() in line 185 as it always is returning None.